syncano/models/custom_sockets_utils.py

```diff
@@ -0,0 +1,222 @@
+# -*- coding: utf-8 -*-
+import six
+from syncano.exceptions import SyncanoValueError
+
+from .incentives import Script
+
+
+class CallTypeE(object):
```

Comment:
Why CallTypeE? To indicate it's an enum? That's internal but it seems a little confusing.
Wouldn't it be enough to do:
```
class CallType:
    SCRIPT = 'script'
```
?

Same for dependencytype